.gitignore

```diff
@@ -12,3 +12,9 @@ manifest-railroad.html
 shell/build
 shell/test/errorShots
 devtools/deps
+host/*
+asylo/bazel-*
```

Comment:
I wondered whether perhaps we should place all of the host-related directories and docs under a top level `host` directory. And then the `.gitignore` fiddling can be specific to that subdir, `Dockerfile` would be under there, it'd be more apparent that asylo relates only to that, etc. OK to leave as is for now tho.